.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "editor.insertSpaces": false
+}
```

Comment:
Could you remove this file please?

I will add `.vscode`  in the .gitignore to avoid this issue.